src/script/services/manifest.ts

```diff
@@ -269,6 +288,7 @@ async function getManifest(): Promise<Manifest | undefined> {
 
 async function generateManifest(url: string): Promise<ManifestDetectionResult> {
   generated = true;
+  sessionStorage.setItem(PWABuilderSession.manifestGenerated, 'true');
```

Comment:
not happy this fires everytime, I'm going to put this in a if statement with the line above, but with the future PR

Reply:
We could also either wrap it in a promise or requestIdleCallback to make sure it does not cause jank (technically sessionStorage is a sync API, so if ALOT is going on in the main thread it can jank things).